frontend/app/src/components/Sidebar/SidebarNav.test.tsx

```diff
@@ -139,80 +128,273 @@ describe("SidebarNav", () => {
     expect(viewButton).not.toBeInTheDocument()
   })
 
```

Comment:
Can maybe add a test for the scenario of 12 pages (all pages displayed, no View button)